Template/LO/Source/PDF/gammaUPC/ebeam.inc

```diff
@@ -0,0 +1,2 @@
+   ebeam(1)=ebeamMG5(1)/(nb_proton(1)+nb_neutron(1))
+   ebeam(2)=ebeamMG5(2)/(nb_proton(2)+nb_neutron(2))
```

Comment:
1) need to check convention is the same for other ion beam (will do it)
2) why having a .inc here?  This sounds weird to me
3) one need to be carefull with the ebeam/ebeamMG5 name here (I do not like the ebeamMG5  name since both are MG5)